src/main/java/org/prebid/server/handler/CookieSyncHandler.java

```diff
@@ -62,6 +61,7 @@
     private final AnalyticsReporter analyticsReporter;
     private final Metrics metrics;
     private final TimeoutFactory timeoutFactory;
+    private final Collection<String> allActiveBidders;
 
     public CookieSyncHandler(String externalUrl, long defaultTimeout, UidsCookieService uidsCookieService,
                              BidderCatalog bidderCatalog, List<List<String>> listOfCoopSyncBidders,
```

Comment:
1. Rename `allActiveBidders` to `activeBidders` and place it after `bidderCatalog`.
2. Pls set `listOfCoopSyncBidders` argument after `defaultCoopSync`.
3. then it can be initialized like: 
```
this.listOfCoopSyncBidders = CollectionUtils.isNotEmpty(listOfCoopSyncBidders) 
  ? listOfCoopSyncBidders 
  : allActiveBidders;
```
It is good practice to make those collections unmodified.